[19.0][MIG] mail_message_search: Migration to 19.0

bizzappdev opened the pull request:
As in Odoo, the record_name field is not a stored field anymore. We have added stored true for the record_name field. TBD: Should we remove the search functionality for record_name? Or have it stored=True, in this case, we should already include the migration script and the init hook, in case we are considering record_name as a searchable field.

AungKoKoLin1997 committed:
[ADD] mail_message_search

Commit:
[UPD] Update mail_message_search.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: social-16.0/social-16.0-mail_message_search
Translate-URL: https://translation.odoo-community.org/projects/social-16-0/social-16-0-mail_message_search/it/

AungKoKoLin1997 committed:
[IMP] mail_message_search: pre-commit auto fixes

AungKoKoLin1997 committed:
[MIG] mail_message_search: Migration to 18.0

Commit:
[UPD] Update mail_message_search.pot

OCA-git-bot committed:
[BOT] post-merge updates

AungKoKoLin1997 committed:
[FIX] mail_message_search : prevent db crash

Co-authored-by: Florent de Labarre <florent.mirieu@gmail.com>

yostashiro commented on mail_message_search/models/mail_message.py:
```diff
@@ -0,0 +1,8 @@
+    _inherit = "mail.message"
+
+    # Inherit field for update the store parameter.
+    record_name = fields.Char(store=True)
```
IMO, we can drop the search on `record_name` as `subject` likely already includes that content in most cases.

yostashiro reviewed (commented):
Would you consider including this commit? https://github.com/OCA/social/pull/1704/commits/6f374ca0305dfc8d312aff23b3fb310d20e8a9c1

yostashiro reviewed (approved):
Code review. LGTM.

> Would you consider including this commit? https://github.com/OCA/social/commit/6f374ca0305dfc8d312aff23b3fb310d20e8a9c1

@bizzappdev In case you've missed this.

AungKoKoLin1997 reviewed (approved):
Functional Review: LGTM

Commit:
[UPD] Update mail_message_search.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: mail-18.0/mail-18.0-mail_message_search
Translate-URL: https://translation.odoo-community.org/projects/mail-18-0/mail-18-0-mail_message_search/

yostashiro reviewed (approved):
LGTM. Thanks!

AungKoKoLin1997 committed:
[FIX] mail_message_search: Error a bytes-like object is required, not str

OCA-git-bot committed:
[BOT] post-merge updates

AungKoKoLin1997 commented:
@bizzappdev Can you please include this fix in your PR?
https://github.com/OCA/mail/pull/103

bizzappdev committed:
[IMP] mail_message_search: pre-commit auto fixes

bizzappdev committed:
[MIG] mail_message_search: Migration to 19.0

AungKoKoLin1997 reviewed (approved):
LGTM

yajo reviewed (approved):
/ocabot merge nobump

yajo commented:
/ocabot migration mail_message_search
